.github/workflows/ocaml-ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ocaml-test:
+    name: Ocaml tests
+    runs-on: ubuntu-20.04
+    env:
+      package: "crc"
+
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Pull configuration from xs-opam
+        run: |
+          curl --fail --silent https://raw.githubusercontent.com/xapi-project/xs-opam/master/tools/xs-opam-ci.env | cut -f2 -d " " > .env
+
+      - name: Load environment file
+        id: dotenv
+        uses: falti/dotenv-action@v0.2.7
+
+      - name: Use ocaml
+        uses: ocaml/setup-ocaml@v1
```

Comment:
What is the difference between `avsm/setup-ocaml@v2` and `ocaml/setup-ocaml@v1`?

Reply:
v2 has some caching enhancements, and needs to add less boilerplate. But it's failing when it tries installing depext on opam 2.1.0, which it shouldn't do